src/WebJobs.Script/Description/DotNet/DotNetFunctionInvoker.cs

```diff
@@ -187,70 +185,38 @@ private void RestorePackages()
                 });
         }
 
-        public override async Task Invoke(object[] parameters)
+        protected override async Task InvokeFunction(object[] parameters, ExecutionContext context, TraceWriter traceWriter,
```

Comment:
You might consider adding an FunctionInvocationContext to capture all these parameters, e.g. Task InvokeCore(FunctionInvocationContext)